public/locales/en/common.json

```diff
@@ -31,7 +31,7 @@
     "CANCEL": "Cancel",
     "ARE_YOU_SURE_DELETE": "There are {{block}} Active blocks in this district you cannot delete it.",
     "NO_ACTIVE_BLOCKS_DELETE": "There are no active blocks in this district. Do you want to delete it?",
-    "ARE_YOU_SURE_DELETE_BLOCK": "There are Active centers in this block you cannot delete it",
+    "ARE_YOU_SURE_DELETE_BLOCK": "There are {{centers}} Active centers in this block you cannot delete it",
```

Comment:
@rahulg1254 change {{centers}} to {{centerCount}} and Active to active

Reply:
ok